src/renderers/shaders/ShaderChunk/lights_physical_pars_fragment.glsl.js

```diff
@@ -110,14 +114,35 @@ void RE_IndirectSpecular_Physical( const in vec3 radiance, const in vec3 clearCo
 		float clearCoatDHR = 0.0;
 	#endif
 
-	reflectedLight.indirectSpecular += ( 1.0 - clearCoatDHR ) * radiance * BRDF_Specular_GGX_Environment( geometry, material.specularColor, material.specularRoughness );
+  float clearCoatInv = 1.0 - clearCoatDHR;
+
+  // Both indirect specular and diffuse light accumulate here
+  // if energy preservation enabled, and PMREM provided.
+  #if defined( ENVMAP_TYPE_CUBE_UV )
+
+    vec3 singleScattering = vec3(0.0);
```

Comment:
minor nit... white space code style. multiple instances in this PR. I know it takes awhile to get used to. :)

Reply:
Oops, looks like `./node_modules/eslint/bin/eslint.js --fix [file]` didn't correct the string glsl which makes sense now -- will fix :+1: